functions/src/modules/SE.functions.ts

```diff
@@ -88,3 +88,58 @@ export const processNewAllotment = functions.database
     });
     return 1;
   });
+
+const serialRegex = "^[a-zA-Z]+-\\d+";
+
+const updateChunk = async (path: string, taskId: string) => {
+  return await db.ref(path).update({
+    taskId
+  });
+};
+
+const makeTaskId = (fileName: string, index: number) => {
+  return `${fileName.match(serialRegex)[0]}-${index}`;
+};
+
+export const createTaskFromChunks = functions.database.ref(
+    "/sound-editing/chunks/{listId}/{fileName}/{index}"
+).onCreate(async (snapshot, { params: { fileName, listId, index } }) => {
+  const {
+    continuationTo,
+    continuationFrom,
+    processingResolution = "",
+    taskId,
+    beginning,
+    ending
+  } = snapshot.val();
+  if (continuationTo || processingResolution.toLowerCase() !== "ok" || taskId) return;
```

Comment:
Here we cannot blindly ignore chunks with `continuationTo`. What if the “continuation” chunk was added first, and this chunk was added after that? We should check whether continuation chunk already exists in the database, and if so — we should create the task from these two chunks.

Reply:
However check that there is no task created from that “continuation” chunk till the date.